libs/native/libraries/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import org.elasticsearch.gradle.transform.UnzipTransform
+
+configurations {
+  libs {
+    attributes.attribute(ArtifactTypeDefinition.ARTIFACT_TYPE_ATTRIBUTE, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+  runtimePath
+}
+
+var zstdVersion = "1.5.5"
```

Comment:
Can we put this in `version.properties`?

Reply:
I don't think we should. version.properties IMO should be where dependencies that need to be aligned across projects should go. Yet zstd is local to this project, we don't need it anywhere else. IMO we should be removing things from version.properties because it gives the false sense that we should be aligning everything when it's both not necessary in most cases, and tends to force transitive dependency versions that were not meant to be used.